### ISSUE TITLE
Rustdoc: crate titles are prefixed with a hash

### ISSUE THREAD

**Issue by stjepang** [rust-lang/rust#39941]
Example: https://doc.rust-lang.org/std/?search=std

Top displayed result:
```
std                 # The Rust Standard Library
```

What I expected:
```
std                 The Rust Standard Library
```

This extra `#` character probably comes from the markdown heading prefix.
In any case, it definitely looks out of place and should not be displayed.

**Comment by frewsxcv** [comment]
Relevant https://github.com/rust-lang/rust/issues/32040

**Comment by ollie27** [comment]
This was fixed by #38330: https://doc.rust-lang.org/nightly/std/?search=std.
